cmd/all-in-one/main.go

```diff
@@ -119,8 +119,14 @@ by default uses only in-memory database.`,
 			aOpts := new(agentApp.Builder).InitFromViper(v)
 			repOpts := new(agentRep.Options).InitFromViper(v, logger)
 			grpcBuilder := agentGrpcRep.NewConnBuilder().InitFromViper(v)
-			cOpts := new(collectorApp.CollectorOptions).InitFromViper(v)
-			qOpts := new(queryApp.QueryOptions).InitFromViper(v, logger)
+			cOpts, err := new(collectorApp.CollectorOptions).InitFromViper(v)
+			if err != nil {
+				logger.Fatal("Incorrect value of flag tls.enabled", zap.Error(err))
```

Comment:
the message should be "Failed to initialize collector", because we do not know the reason why it failed (it will be contained in the `err` which will be printed separately

Reply:
yeah i was thinking the same thanks :)